phase1-coordinator/src/lib.rs

```diff
@@ -35,3 +35,6 @@ pub mod testing;
 
 #[cfg(test)]
 pub mod tests;
+
+#[macro_use]
+extern crate serde_with;
```

Comment:
try to do away with this global macro import if possible

Reply:
it kind of feels controversial to
https://github.com/AleoHQ/aleo-setup/pull/242#issuecomment-814795389

Reply:
that's right, it was a quick hack to get it to work, but I found the fix now to remove this global import, it was pretty easy in the end